### PR TITLE
Completes basic file load test locally

### PR DESCRIPTION
**Pull request by LukeLoze1**

### CODE REVIEW

File: utilities/action_service.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import requests
+import tzlocal
+
+from config import Config
+
+
+def create_action_plan(survey_ref, collection_exercise_id):
+    try:
+        dt = convert_to_iso_timestamp('2019-02-11T11:11')
```
Comment (rjweeks) [inline utilities/action_service.py]: Use date_utilities.get_timestamp_with_offset

File: utilities/action_service.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    return action_plans_response.json()
+
+
+def plan_for_collection_exercise(plan, collection_exercise_id):
+    if not plan['selectors']:
+        return False
+    return plan['selectors']['collectionExerciseId'] == collection_exercise_id
+
+
+def convert_to_iso_timestamp(timestamp_string):
```
Comment (rjweeks) [inline utilities/action_service.py]: Move this to date_utilities

**Review by rjweeks — CHANGES_REQUESTED** [review]

File: Pipfile [hunk 3]
```diff
@@ -20,9 +20,13 @@ paramiko = "*"
 pytz = "*"
 tzlocal = "*"
 psutil = "*"
+v = {editable = true,version = "*"}
```
Comment (rjweeks) [inline Pipfile]: I don't think this is required. Please delete and re-test.

File: acceptance_tests/features/steps/setup_steps.py [hunk 4]
```diff
@@ -40,8 +41,8 @@ def there_is_a_live_collex(context, unique_id):
     mandatory_events = {
         "mps": "2019-02-08T15:00:00.000Z",
         "go_live": "2019-02-08T16:00:00.000Z",
-        "return_by": "2019-02-08T17:00:00.000Z",
-        "exercise_end": "2019-02-08T18:00:00.000Z"
+        "return_by": "2019-02-18T17:00:00.000Z",
+        "exercise_end": "2019-02-18T18:00:00.000Z"
     }
```
Comment (rjweeks) [inline acceptance_tests/features/steps/setup_steps.py]: I have raised a PR that replaces these hard-coded dates with get_timestamp_with_offset from date_utilities.

**Review by rjweeks — CHANGES_REQUESTED** [review]

File: acceptance_tests/features/steps/setup_steps.py (hunk 4 above)
Comment (LukeLoze1) [reply acceptance_tests/features/steps/setup_steps.py]: This PR was raised prior to that

File: utilities/action_service.py (hunk 1 above)
Comment (LukeLoze1) [reply utilities/action_service.py]: current version doesn't do days

Comment (LukeLoze1) [reply utilities/action_service.py]: I've hardcoded the date for now until the correct offset function exists

File: utilities/action_service.py (hunk 2 above)
Comment (LukeLoze1) [reply utilities/action_service.py]: removed altogether.  It clashes in data_utils different datetime imports

**Comment by rjweeks** [comment]
[Automated testing](https://trello.com/c/PUT3GUaJ/438-automated-testing)

**Review by rjweeks — APPROVED** [review]

File: utilities/action_service.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    action_plans_response.raise_for_status()
+    return action_plans_response.json()
+
+
+def plan_for_collection_exercise(plan, collection_exercise_id):
+    if not plan['selectors']:
+        return False
+    return plan['selectors']['collectionExerciseId'] == collection_exercise_id
+
+
```
Comment (hbrace) [inline utilities/action_service.py]: Lot's of blank space in this file

File: utilities/collection_exercise_utilities.py [hunk 6]
```diff
@@ -66,7 +66,7 @@ def create_mandatory_events(collection_exercise_id, collex_events: dict):
             'timestamp': timestamp
         }
         response = requests.post(collex_event_url, auth=Config.BASIC_AUTH, json=event_tag)
-        response.raise_for_status()
+        #response.raise_for_status()
```
Comment (hbrace) [inline utilities/collection_exercise_utilities.py]: Leave raise_for_status() in or remove comment?

**Review by hbrace — CHANGES_REQUESTED** [review]

**Review by hbrace — APPROVED** [review]

**Pull request merged**